docs/getting-started/fundamentals-block-development/file-structure-of-a-block.md

```diff
@@ -0,0 +1,88 @@
+# File structure of a block
+
+The most common way to register a custom block is via a JSON file that contains all of the metadata of the block.
```

Comment:
This line feels out of place. I am not sure how it directly relates to the file structure.  🤔